domain/fizz_buzz/fizz_buzz.py

```diff
@@ -7,12 +7,14 @@ def is_multiple_of_three(a_number):
 
 
 def map_from_integer_to_fizz_or_buzz(a_number):
+    accumulated_result = ""
     if is_multiple_of_three(a_number):
-        return "fizz"
+        accumulated_result = "fizz"
```

Comment:
I'd suggest to add to previous `accumulated_result`
```suggestion
        accumulated_result += "fizz"
```